modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -373,6 +373,9 @@ const NotificationsInfo = ({
 				else if (recipientType === 'taskAssignees') {
 					recipientDetails = {assignmentType: ['taskAssignees']};
 				}
+				else if (recipientType === 'scriptedRecipient') {
+					recipientDetails = {scriptLanguage: ['groovy']};
+				}
```

Comment:
Hi @Marina-Gouveia, sort this conditional, it should be before [taskAssignees](https://github.com/liferay-workflow/liferay-portal/pull/1013/files#diff-aeacf12a0c6bf7decf8d6bc07c1a7d0e1e6b9800bf3676665664fc379371e830R373).

Thanks!

Reply:
Hi @MarinhoFeliphe, done!!